Formula/clang-format.rb

```diff
@@ -32,15 +32,6 @@ class ClangFormat < Formula
     regex(%r{href=.*?/tag/llvmorg[._-]v?(\d+(?:\.\d+)+)}i)
   end
 
-  bottle do
```

Comment:
Please put back the `bottle` block. You're not meant to touch this.

Reply:
Sorry, was under the assumption that it will need to be recreated in full by the bot, as of

> Otherwise, a bottle block will be added by Homebrew’s CI upon merging the formula’s pull-request.

Undid the removal 👍